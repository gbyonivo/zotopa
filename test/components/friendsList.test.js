import React from 'react';
import renderer from 'react-test-renderer';
import FriendList from '../../src/components/friendsList';

describe('FriendList', () => {
  it('should render correctly', () => {
    const actual = renderer.create(<FriendList friends={[{ name: 'ivo', id: 1 }]}/>);
    expect(actual).toMatchSnapshot();
  });
  it('should render correctly with empty friends', () => {
    const actual = renderer.create(<FriendList friends={[]}/>);
    expect(actual).toMatchSnapshot();
  });
});