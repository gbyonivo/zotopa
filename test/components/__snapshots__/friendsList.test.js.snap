// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FriendList should render correctly 1`] = `
<ul>
  <div>
    ivo
  </div>
</ul>
`;

exports[`FriendList should render correctly with empty friends 1`] = `
<div>
  No Friends
</div>
`;
